ydb/library/workload/clickbench/clickbench.cpp

```diff
@@ -164,6 +173,8 @@ void TClickbenchWorkloadParams::ConfigureOpts(NLastGetopt::TOpts& opts, const EC
             .StoreResult(&ExternalQueries);
         opts.AddLongOption('c', "check-canonical", "Use deterministic queries and check results with canonical ones.")
             .NoArgument().StoreTrue(&CheckCanonicalFlag);
+        opts.AddLongOption( "pg-syntax", "Use postgree syntax in queries instead YQL one.")
```

Comment:
We should probably use "--syntax pg" option with default value "yql". We already have such option in ydb sql command, would be consistent with it. Also would be easier to support new syntaxes, if they will appear